apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.ts

```diff
@@ -12,6 +12,19 @@ import { InstructorSharing } from './instructorCourseAdminSharing.html.js';
 const router = Router();
 const sql = sqldb.loadSqlEquiv(import.meta.url);
 
+async function selectCanChooseSharingName(course) {
+  return (
+    course.sharing_name === null ||
+    !(await sqldb.queryOptionalRow(
+      sql.select_shared_question_exists,
+      {
+        course_id: course.id,
+      },
+      z.any().nullable(),
```

Comment:
Why is this typed as `z.any()`? Will `SELECT EXISTS ...` not return a boolean?

Reply:
oops, ya that should definitely be `z.boolean()`, sorry for missing that in my review.

Reply:
Changed to z.boolean()